helpers/polizasHelper/polizashelper.go

```diff
@@ -0,0 +1,260 @@
+	)
+
+	//Traer el detalle de las entradas (json)
+	params := url.Values{}
+	params.Add("query", "TipoBienId__NecesitaPoliza:True,Activo:True")
+	params.Add("limit", "-1")
+	urlSubgrupos := "http://" + beego.AppConfig.String("catalogoElementosService") + "detalle_subgrupo?" + params.Encode()
+	if _, err := request.GetJsonTest(urlSubgrupos, &Respuesta); err != nil {
+		logs.Error(err)
+		outputError = map[string]interface{}{
+			"funcion": "GetSubgruposPoliza - request.GetJsonTest(urlSubgrupos, &Respuesta)",
+			"err":     err,
+			"status":  "502",
+		}
+		return nil, outputError
+	}
+
+	// Asignar a DetalleSubgrupo RespuestaAPI
+	if err := formatdata.FillStruct(&Respuesta, &DetalleSubgrupo); err != nil {
+		logs.Error(err)
+	}
+
+	// Itera los DetalleSubgrupo para guardar solo los SubgrupoId
+	for _, detallesub := range DetalleSubgrupo {
+		IdDetalleSubgrupo = append(IdDetalleSubgrupo, detallesub.SubgrupoId.Id)
+	}
+
+	// Remueve los subgrupos replicados
+	IdSubgruposPoliza = removeDuplicateIds(IdDetalleSubgrupo)
+
+	return
+}
+
+// Consulta los elementos por ActaId y retorna algunos parametros
+//
+func GetElementosPolizas(ActasIdsEntradas []int, SubgrupoPoliza []int, limit int, offset int, fields []string, order []string, query map[string]string, sortby []string) (ElementosEntradas *[]models.Elemento, outputError map[string]interface{}) {
+
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{
+				"funcion": "GetElementosPorActaIds - Unhandled Error!",
+				"err":     err,
+				"status":  "500",
+			}
+			panic(outputError)
+		}
+	}()
+
+	var (
+		Respuest interface{}
+		Query    []string
+		QUERY    string
+	)
+
+	// Se convierten los filtros en terminos de string para poder hacer la consulta por url
+	lim := strconv.Itoa(limit)
+	off := strconv.Itoa(offset)
+	field := strings.Join(fields, ",")
+	sort := strings.Join(sortby, ",")
+	orde := strings.Join(order, ",")
+
+	params := url.Values{}
+
+	for key, value := range query {
+		Query = append(Query, key+":"+value)
+		QUER := strings.Trim(strings.Replace(fmt.Sprint(Query), " ", ",", -1), "[]")
+		QUERY = QUER
+	}
+
+	if QUERY != "" {
+		params.Add("query", "Activo:True,"+QUERY+",ActaRecibidoId__Id__in:"+utilsHelper.ArrayToString(ActasIdsEntradas, "|")+",SubgrupoCatalogoId__in:"+utilsHelper.ArrayToString(SubgrupoPoliza, "|"))
+	} else {
+		params.Add("query", "Activo:True,ActaRecibidoId__Id__in:"+utilsHelper.ArrayToString(ActasIdsEntradas, "|")+",SubgrupoCatalogoId__in:"+utilsHelper.ArrayToString(SubgrupoPoliza, "|"))
+	}
+
+	if lim != "10" {
+		params.Add("limit", lim)
+	}
+	if off != "0" {
+		params.Add("offset", off)
+	}
+	if field != "" {
+		params.Add("fields", field)
+	}
+	if sort != "" {
+		params.Add("sortby", sort)
+	}
+	if orde != "" {
+		params.Add("order", orde)
+	}
+
+	urlElementosSubgrupos := "http://" + beego.AppConfig.String("actaRecibidoService") + "elemento?" + params.Encode()
+	logs.Debug(urlElementosSubgrupos)
```

Comment:
(En general) Dejar los `logs.Debug()` comentados y/o eliminarlos
```suggestion
	// logs.Debug(urlElementosSubgrupos)
```